src/db/index.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const util = require('util');
+const path = require('path');
+
+let readFile = util.promisify(fs.readFile);
+//let writeFile = util.promisify(fs.writeFile);
+
+const productsPath = path.resolve('./src/db/product.json');
+
+const register = (newUser) => {
+  fs.readFile('./src/db/users.json', 'utf8', (err, data) => {
```

Comment:
you're right, I forgot that I wrapped it with a promise. 

The code should look like this with then syntax. Because the promisified function is saved to a variable called readFile, you should drop the fs. prefix. 

With .then() syntax:

`readFile(<path>)
    .then(...);`